.github/workflows/build-and-test.yml

```diff
@@ -173,6 +173,9 @@ jobs:
       - name: Run Unit Tests
         run: |
           make -j4 gotest-with-junit
+      - name: Run Unit Tests with GODEBUG=fips140=only
+        run: |
+          make -j4 gotest-fips140-only
```

Comment:
I'm not familiar with this GODEBUG option, but does it have to be its own run? I'd be nice not to run all the unit tests twice

Reply:
Done in e32b4feb95968c57c127779919bf3dfd2602546e